image-builder/image_builder.sh

```diff
@@ -324,7 +324,11 @@ format_loop() {
 			;;
 
 		"${xfs_format}")
-			mkfs.xfs -q -f -b size="${block_size}" "${device}p1"
+			# DAX and reflink cannot be used together!
+			# Explicitly disable reflink, if it fails then reflink
+			# is not supported and '-m reflink=0' is not needed.
+			mkfs.xfs -m reflink=0 -q -f -b size="${block_size}" "${device}p1" \
```

Comment:
Hmm. What's the error reported here in case reflink is not supported?

What I'm worried is that we'd face a real error, ignore it, and just go to the second part of the expression ... totally ignoring the first valid error.

Does this make sense?

Reply:
got it, but how I can detect whether the distro supports `reflink` or not? in distros without `reflink` support `mkfs.xfs -m reflink=0 ..` fails

Reply:
How does it fail? Any specific error on the failure?